packages/app-builder/src/components/hooks/objects/use-sub-objects.ts

```diff
@@ -0,0 +1,33 @@
+import { useIntrospection } from '../use-introspection'
+import { usePageId } from '../use-page-id'
+
+export const lowercaseFirstLetter = (string: string) => string.charAt(0).toLowerCase() + string.slice(1)
+
+export const useSubObjects = (typeName?: string) => {
```

Comment:
Add a return type here? I can see false, {data: T[], error, loading}? and true? so `SubObject | boolean`? Not entrely sure